lib/standard/string.nit

```diff
@@ -849,6 +849,38 @@ abstract class Text
 		return hash_cache.as(not null)
 	end
 
+	# Gives the formatted string back as a Nit string with `args` in place
+	#
+	# 	assert "This %1 is a %2.".unformat("String", "formatted String") == "This String is a formatted String"
+	fun unformat(args: Object...): String do
```

Comment:
I thought that by doing this, we transform a formatted string to a plain one, which is why I called it `unformat`, but if everyone agrees this is a bad name and it should be named `format` instead, I'd change it !